src/test/regress/expected/multi_view.out

```diff
@@ -486,17 +486,73 @@ ORDER BY 2 DESC, 1;
        5 | NO
 (3 rows)
 
--- event vs table non-partition-key join is not supported
--- given that we cannot recursively plan tables yet
+-- event vs table non-partition-key join is supported
+-- given that we can recursively plan events_table
 SELECT * FROM
```

Comment:
minor: maybe change to SELECT count(*),user_id,done_event .. GROUP BY user_id,done_event ORDER BY 1,2,3